src/__tests__/render.js

```diff
@@ -113,3 +113,44 @@ test('renders options.wrapper around node', () => {
 </div>
 `)
 })
+
+describe('baseElement', () => {
+  let baseElement
+  beforeAll(() => {
+    baseElement = document.createElement('div')
+    document.body.appendChild(baseElement)
+  })
+
+  afterAll(() => {
+    baseElement.parentNode.removeChild(baseElement)
+  })
+
+  it('can take a custom element for isolation', () => {
+    function DescribedButton({title: description, ...buttonProps}) {
```

Comment:
Is it necessary that this component be so complex? I prefer to make the component simpler/contrived to get the point across. I believe that this test would pass with or without the `baseElement`... Can we have a test that actually requires the `baseElement`?

Reply:
I'll give it a try to make this more meaningful. It does, however, fail without the fix applied or without the baseElement passed.